backend/wmg/data/cube_pipeline.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from typing import List
+
+from backend.wmg.data.extract import copy_datasets_to_instance
+from backend.wmg.data.load import update_s3_resources
+from backend.wmg.data.transform import get_cells_by_tissue_type, generate_cell_ordering
+from tests.unit.backend.wmg.fixtures.cube import create_cube
+
+
+def load(dataset_directory: List, group_name: str, validate: bool):
+    """
+    Given the path to a directory containing one or more h5ad files and a group name, call the h5ad loading function
+    on all files, loading/concatenating the datasets together under the group name
+    """
+    pass
+
+
+def load_data_and_create_cube(path_to_datasets: str, corpus_name: str, log_level):
+    """
+    Function to copy H5AD datasets (from a preconfiugred s3 bucket) to the path given then,
+    open, transform, normalize and concatenate them together as a tiledb object with a global gene index
+    under the given corpus name.
+    A cube of summary statistics across genes (but queryable by the given dimensions/data) is then generated
+    and stored under big-cube.
```

Comment:
In API layer, I'm referring to this cube as the "expression summary" cube, to differentiate from the upcoming "cell counts" cube.